nengo/config.py

```diff
@@ -139,16 +133,13 @@ def __init__(self, configures, clsparams):
     def __contains__(self, key):
         return self in self._clsparams.get_param(key)
 
-    def __getattribute__(self, key):
-        try:
-            return super(InstanceParams, self).__getattribute__(key)
-        except AttributeError:
-            if key in self._clsparams.default_params:
-                raise
-            param = self._clsparams.get_param(key)
-            if self in param:
-                return param.__get__(self, self.__class__)
-            return getattr(self._clsparams, key)
+    def __getattr__(self, key):
+        if key in self._clsparams.default_params:
+            raise AttributeError()
```

Comment:
But even if I comment out the first line in your example it still doesn't work, because of the AttributeError in `InstanceParams.__setattr__`. I'm not totally clear on how `InstanceParams` is supposed to work.